src/fatfs/ffconf.h

```diff
@@ -66,6 +59,33 @@
 /* This option switches f_forward() function. (0:Disable or 1:Enable) */
 
 
+// FIXME: remove this, do not merge to stable without further discussion

```

Comment:
Why are you adding this FIXME here?

Reply:
I am not. That is your comment in unstable.

Reply:
https://github.com/DragonMinded/libdragon/blob/352679922bad5403f8ea4b9123dfe0403d030bc2/src/fatfs/ffconf.h#L27

Reply:
oh i had forgotten about that one. Do you still need that #ifdef?

Reply:
Feel free to remove it. If needed I can workaround using SED before build.